sport/app/rugby/feed/OptaFeed.scala

```diff
@@ -24,7 +19,7 @@ case object WorldCup2015 extends OptaEvent {
 
 case class RugbyOptaFeedException(message: String) extends RuntimeException(message)
 
-object OptaFeed extends ExecutionContexts with Logging {
+case class OptaFeed(wsClient: WSClient) extends ExecutionContexts with Logging {
```

Comment:
I think a `class` would be more appropriate than a `case class` here as its only state is the `wsClient.